### DISCUSSION TITLE
SEB error suppression

### DISCUSSION THREAD

**Discussion by fkm97** [SafeExamBrowser/seb-win-refactoring#693] [Q&A]
Is there a way on SEB to prevent this error popup? I allowed this application to be used in SEB but it cannot find it (which is fine). I just want this error to not popup. For more clarity, I mean if the application is available on the machine, all well and good. Continue as is. If it is not available, have no error pop up and just continue as is. If i do click OK, its fine and it will continue as is. 

![20231025_145107](https://github.com/SafeExamBrowser/seb-win-refactoring/assets/72949739/a1beec98-3ec4-4a8a-a17c-4609f0b4e2cd)

**Comment by dbuechel** [comment]
There already exists a feature request regarding such a configuration option, please refer to #572.
